ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -174,6 +179,25 @@ open class PillButton: UIButton {
         setNeedsLayout()
     }
 
+    @objc private func titleValueDidChange() {
+        if #available(iOS 15.0, *) {
+            updateAttributedTitle()
+        } else {
+            setTitle(pillBarItem.title, for: .normal)
+        }
+
+    }
+
+    @available(iOS 15, *)
+    private func updateAttributedTitle() {
+        var attributedTitle = AttributedString(pillBarItem.title)
+        attributedTitle.font = Constants.font
+        configuration?.attributedTitle = attributedTitle
```

Comment:
I believe there is also an iOS 15+ issue with VoiceOver and the PillButton where the selected item wouldn't read the button title as the accessibility description/label. It's as if the UIButton.Configuration.attributedTitle isn't propagated to the accessibility property as it is should be. Could the accessibility value be set here to resolve this issue?

Reply:
will check this

Reply:
i have a separate bug for this. can you update your branch first and merge? i can take a look

Reply:
Rebased on main to include your fix. Relocated it to the update function because it needs to rerun on every title change.